supervisor/validate.py

```diff
@@ -148,7 +148,10 @@ def validate_repository(repository: str) -> str:
         vol.Optional(ATTR_IMAGE): docker_image,
         vol.Optional(
             ATTR_ADDONS_CUSTOM_LIST,
-            default=["https://github.com/hassio-addons/repository"],
+            default=[
+                "https://github.com/hassio-addons/repository",
+                "https://github.com/esphome/hassio",
+            ],
```

Comment:
This will only apply if the restored data have a blank list?
And since we have "https://github.com/hassio-addons/repository" already this will never be blank?

Reply:
That is for now all what we can do until someone finish the store code

Reply:
Right, so this will only be there on new installations.

Reply:
So, if there is ever an oppertunaty to rename `hassio` in the repo name of ESPHome to `addons` or something similar, this is the time!

Reply:
renaming the repo is transparent as GitHub will redirect to the new and this URL is not shown to the user so I don't think that is needed before this right?

Reply:
> renaming the repo is transparent as GitHub will redirect to the new and this URL is not shown to the user so I don't think that is needed before this right?

That is not forever, and will change the hash.

Reply:
yeah, I don't have esphome org permissions